scripts/explain-warn.sh

```diff
@@ -0,0 +1,62 @@
+#!/bin/sh
+
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+echo 'Creating 10,000 dummy accounts, hang tight...'
+node scripts/populate-session-tokens 10000 2>&1 > /dev/null
+
+SRC="lib/db/mysql.js"
+ACTIVE_PROCEDURES=`grep 'CALL ' "$SRC" | awk -F 'CALL +' '{print $2}' | cut -d '(' -f 1`
+
+for PROCEDURE in $ACTIVE_PROCEDURES
+do
+  FILE=`git grep "$PROCEDURE" | grep 'CREATE PROCEDURE' | cut -d ':' -f 1 | sed '/^$/d'`
+  EXPLAINS=`node scripts/explain-queries "$FILE" "$PROCEDURE"`
+
+  if [ "$EXPLAINS" = "" ]
+  then
+    continue
+  fi
+
+  echo "$EXPLAINS" | while read -r EXPLAIN
+  do
+    EXPLAIN_RESULT=`mysql -D fxa -u root -e "$EXPLAIN" 2>/dev/null`
+    EXPLAIN_DATA=`echo "$EXPLAIN_RESULT" | grep -v 'id\tselect_type'`
```

Comment:
I've never used it, by MySQL lets you output explain results in JSON format using `EXPLAIN FORMAT=JSON SELECT ...`.  Combined with some switches to disable pretty-printed output, you can use this to output the explain result as a properly formatted JSON file, which may be easier to work with than grepping the string outputs.  Example:

```
$> mysql -BNr -D fxa -u root -h 127.0.0.1 -e "EXPLAIN FORMAT=JSON SELECT * FROM sessionTokens ORDER BY tokenId";
{
  "query_block": {
    "select_id": 1,
    "ordering_operation": {
      "using_filesort": false,
      "table": {
        "table_name": "sessionTokens",
        "access_type": "index",
        "key": "PRIMARY",
        "used_key_parts": [
          "tokenId"
        ],
        "key_length": "32",
        "rows": 3268,
        "filtered": 100
      }
    }
  }
}
```